pokete.py

```diff
@@ -734,7 +425,7 @@ def __call__(self):
 {obj.attack_dict['name']}'?"):
                                     ex_cond = True
                                     while ex_cond:
-                                        index = deck(6, label="Your deck",
+                                        index = deck.deck(6, label="Your deck",
                                                      in_fight=True)
```

Comment:
Due to adding `deck.` in line 428, this line need to be indented by further five spaces.

```suggestion
                                                          in_fight=True)
```